index.js

```diff
@@ -20,9 +20,23 @@ function codeImport(options = {}) {
         continue;
       }
 
-      const filePath = fileMeta.slice('file='.length);
+      const res = /^file=(?<path>[^#]+)(?:#L(?<from>\d+))?(?::L(?<to>\d+))?$/.exec(fileMeta)
```

Comment:
This will fail to match if the path itself contains `#` somewhere. Could we adjust this so that only the last `#` is accounted?

Reply:
Good catch. Fixed.

Note I also changed the test file to `say-#-hi.js` to test for this.